src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -309,11 +328,12 @@ public function clickLink(string $linkText): Crawler
     /**
      * Submits a form.
      *
-     * @param Form  $form             A Form instance
-     * @param array $values           An array of form field values
-     * @param array $serverParameters An array of server parameters
```

Comment:
this should be kept. We do have this param (as a hidden param handled through `func_get_arg`  due to BC reasons))